content/version/2/0/code_of_conduct.pt-br.md

```diff
@@ -0,0 +1,90 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Código de Conduta para Colaborantes
+
+## Nossa promessa
+
+Nós, como participantes, contribuintes e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
+
+Comprometemo-nos a agir e interagir de maneiras que contribuem para uma comunidade aberta, acolhedora, diversificada, inclusiva e saudável.
+
+## Nossos padrões
+
+Exemplos de comportamentos que contribuem para criar um ambiente positivo para a nossa comunidade incluem:
+
+* Demonstrar empatia e bondade com as outras pessoas
+* Respeitar opiniões contrárias, pontos de vista e experiências
+* Dar e receber feedbacks construtivos de maneira graciosa
```

Comment:
I'd use "respeitosa" instead of "graciosa". I can't think of a good translation for "graceful" in this sentence. "Respeitosa" is "respectful" which is a good free translation in my opinion.

Reply:
Tem razão. Eu adaptei do texto da versão 1 (que dizia "com graça"), mas concordo com você que assim fica melhor. Estou alterando para "respeitosa".